tests/pyoozie/conftest.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
+# Use of this source code is governed by a MIT-style license that can be found in the LICENSE file.
+from __future__ import unicode_literals
+
+import uuid
+
+import mock
+import pytest
+
+from six import moves
+
+
+@pytest.fixture
+def fake_uuid4(monkeypatch):
```

Comment:
Let's test our uid identifiers generated in `_WorkflowEntity`, but reduce their randomness a bit.